tests/foreman/cli/test_role.py

```diff
@@ -426,81 +431,832 @@ def test_positive_update_role_taxonomies(self):
+
+        :CaseLevel: Integration
+        """
+
+    @stubbed
+    @tier2
+    def test_positive_clone_role_without_taxonomies_non_overided_filter(self):
+        """When taxonomies not assigned to cloned role, only unlimited but not
+        override flag sets on filter for filter that is overridden in parent
+        role
+
+        :id: 92264a5f-7cd8-4a91-8089-f2f546f556b3
+
+        :steps:
+
+            1. Create a role with organization A and Location A
+            2. Create role filter without overriding
+            3. Clone above role without assigning taxonomies
+            4. List cloned role filter
+
+        :expectedresults:
+
+            1. Unlimited flag should be set to True
+            2. Override flag should be set to False
+
+        :CaseLevel: Integration
+        """
+
+    @stubbed
+    @tier2
+    def test_positive_clone_role_without_taxonomies_unlimited_filter(self):
+        """When taxonomies not assigned to cloned role, Unlimited and override
+        flags sets on filter for filter that is unlimited in parent role
+
+        :id: 2f205923-f590-4797-b63b-adf389f802e6
+
+        :steps:
+
+            1. Create a role with organization A and Location A
+            2. Create role filter with unlimited check
+            3. Clone above role without assigning taxonomies
+            4. List cloned role filter
+
+        :expectedresults:
+
+            1. Unlimited flag should be set to True
+            2. Override flag should be set to False
+
+        :CaseLevel: Integration
+        """
+
+    @stubbed
+    @tier2
+    def test_positive_force_unlimited(self):
+        """Unlimited flag forced sets to filter when no taxonomies are set to role
+        and filter
+
+        :id: 2de03e36-7f8d-4b17-819a-0d3b4468e32c
+
+        :steps:
+
+            1. Create a role with organization A and Location A
+            2. Create a role filter without override and unlimited check
+            3. Remove taxonomies assigned earlier to role and ensure
+                no taxonomies are assigned to role
+            4. List Role filter
+
+        :expectedresults: Unlimited flag should be forcefully set on filter
+            when no taxonomies are set to role and filter
+
+        :CaseLevel: Integration
+        """
+
+    @stubbed
+    @tier3
+    def test_positive_user_group_users_access_as_org_admin(self):
+        """Users in usergroup can have access to the resources in taxonomies if
+        the taxonomies of Org Admin role is same
+
+        :id: 630fcd05-5c27-44a7-9bea-fcef1143b252
+
+        :steps:
+
+            1. Create an Org Admin role by cloning 'Organization admin' role
+            2. Assign an organization A and Location A to the Org Admin role
+            3. Create two users without assigning roles while creating them
+            4. Assign Organization A and Location A to both users
+            5. Create an user group with above two users
```

Comment:
It is not about taxonomies but it is about assigning org admin role to usergroup.Also all users in usergroup able to access all the resources as Admin of that org. Its very valid scenario